elements/pl-integer-input/pl-integer-input.py

```diff
@@ -37,9 +37,22 @@ def prepare(element_html, data):
         if name in data['correct_answers']:
             raise Exception('duplicate correct_answers variable name: %s' % name)
         # Test conversion, but leave as string so proper value is shown on answer panel
-        if pl.string_to_integer(correct_answer, base) is None:
+        a_tru_parsed = pl.string_to_integer(correct_answer, base)
+        if a_tru_parsed is None:
             raise Exception('correct answer is not a valid input: %s' % name)
+        if a_tru_parsed > 2 ** 53 - 1 or a_tru_parsed < -((2 ** 53) - 1):
```

Comment:
Generally prefer no spaces around exponentiation operator:
```suggestion
        if a_tru_parsed > 2**53 - 1 or a_tru_parsed < -((2**53) - 1):
```